Rubberduck.Parsing/ParserRuleContextExtensions.cs

```diff
@@ -173,6 +174,25 @@ private static TContext GetAncestor_Recursive<TContext>(ParserRuleContext contex
             return opCtxt != null;
         }
 
+        /// <summary>
+        /// Determines if the context's module declares or defaults to 
+        /// Option Compare Binary 
+        /// </summary>
+        public static bool IsOptionCompareBinary(this ParserRuleContext context)
```

Comment:
When I wrote this function I was looking at information like this: https://msdn.microsoft.com/en-us/vba/language-reference-vba/articles/option-compare-statement  <- this looks like Option Compare Binary (case sensitive) is the default even for Access.  Am I mis-interpreting this?

Reply:
It is the default, but Access will always add `Option Compare Database` to every new module and class.

Reply:
Not a grammar issue, but when testing `Compare` be aware that the Immediate window has its own `Compare` setting which becomes the default when used in native functions inside the immediate window, regardless of what the setting is in the activr project/module. IIRC there was an SO question/discussion in chat about it.

Reply:
The issue is that `Option Compare Text` and `Option Compare Database` will give different results. AFAIK, the `Option Compare Text` is simply case insensitive and never ever is accent insensitive. With `Option Compare Database`, it may or may not be accent-sensitive, depending on the collating order of the file. This is likely beyond the PR's scope and is also a host-specific thing. I'm sure that `Option Compare Database` is not legal in any other VBA hosts. Nonetheless, because this is going to end up as an expression engine, so it'll be necessary to handle this edge case. We would also need to review VBA specs to see if `Option Compare Text` will likewise behave differently with a non-English host.